app/views/job_templates/index.html.erb

```diff
@@ -2,7 +2,7 @@
 <%= javascript 'foreman_remote_execution/template_input' %>
 
 <% title _("Job Templates") %>
-<% title_actions(documentation_button_rex('3.1JobTemplates'),
+<% title_actions(documentation_button('Managing_Hosts', type: 'docs', chapter: 'creating-a-job-template_managing-hosts'),
```

Comment:
@aneta-petrova This may not be actionable in the moment, but in the old documentation we had a sort of "landing page" for job templates (https://theforeman.org/plugins/foreman_remote_execution/1.7/index.html#3.1JobTemplates). Now we link to https://docs.theforeman.org/3.16/Managing_Hosts/index-katello.html#creating-a-job-template_managing-hosts even though the page also allows importing etc.

Perhaps this is a nice hook into the REX content journey you had in mind.